.github/workflows/pr.yml

```diff
@@ -75,9 +80,9 @@ jobs:
         with:
           version: "2021.1.2"
           install-go: false
-      - name: Check manifests
+      - name: Check all manifests
         run: |
-          make generate manifests
+          CONTROL_PLANE=kcp make generate manifests
```

Comment:
This would ignore anything in the `kubernetes` directory. If rbac is going to be different not sure if it will even work. Webhooks will be disabled too.

Reply:
I changed it to two separate checks. wdyt?

Reply:
:+1: